cmd/bridge/main.go

```diff
@@ -528,7 +528,7 @@ func main() {
 		}
 		srv.InternalProxiedK8SClientConfig.BearerToken = *fK8sAuthBearerToken
 	case "oidc", "openshift":
-		flags.FatalIfFailed(flags.ValidateFlagIs("user-auth", authOptions.AuthType, "oidc", "openshift"))
+		flags.FatalIfFailed(flags.ValidateFlagIs("user-auth", authOptions.AuthType, "oidc", "openshift", "disabled"))
```

Comment:
To be honest, I believe the whole `k8s-auth` flag should be removed. The only value that appears to make sense is `bearer-token`, which in fact replaces your authenticator. That should be its own flag that basically makes the console operate in a non-proxy mode with a hardcoded test-user identified by the token.
From that point, you could just create your own authenticator that uses the hardcoded user (you'll have to change the authenticator struct to interface).
When you get here, you'll be able to determine in the `srv.HTTPHandler()` that you should return a handler that sends you a "No authentication configured" web page.

Reply:
We don't have time to make this big of a change now. This would also change the API, which is out of scope for a bug fix. We could follow up with a tech debt story though.

Reply:
I mean removing the `k8s-auth` flag, not the suggested change.

Reply:
There will never be time, but I can do it if you want.